master/buildbot/www/ldapuserinfo.py

```diff
@@ -113,6 +113,13 @@ def thd():
                               attributes=[
                                   self.accountEmail, self.accountFullName] +
                               self.accountExtraFields)
+            # active directory delivers references in addition to the search
+            # results - filter them out
+            tmp = []
+            if len(res) > 1:
+                for elem in res:
+                    if elem['type'] == 'searchResEntry':
+                        tmp.append(elem)
```

Comment:
do you mean to do res = tmp after the loop?

Reply:
Right <blushes>, seems I was too sloppy when making the patch. Will Fix.

Reply:
Well, now the missing line is... below our conversation and easy to miss... Sorry for that.